docs/index.md

```diff
@@ -289,11 +289,12 @@ Core utilities such as Tracing, Logging, and Metrics will be available across al
 | [Parameters](./utilities/parameters.md)   | High-level functions to retrieve one or more parameters from AWS SSM Parameter Store, AWS Secrets Manager, AWS AppConfig, and Amazon DynamoDB                     |
 | [Idempotency](./utilities/idempotency.md) | Class method decorator, Middy middleware, and function wrapper to make your Lambda functions idempotent and prevent duplicate execution based on payload content. |
 | [Batch Processing](./utilities/batch.md)  | Utility to handle partial failures when processing batches from Amazon SQS, Amazon Kinesis Data Streams, and Amazon DynamoDB Streams.                             |
+| [Parser](./utilities/parser.md)           | Utility to parse and validate AWS Lambda event payloads using Zod, a TypeScript-first schema declaration and validation library.                                  |
 
 ## Environment variables
 
 ???+ info
-	Explicit parameters take precedence over environment variables
+Explicit parameters take precedence over environment variables
```

Comment:
<img width="871" alt="image" src="https://github.com/aws-powertools/powertools-lambda-typescript/assets/7353869/36b734ec-e4f2-461c-bcd1-52f6908a5e16">

should be

<img width="867" alt="image" src="https://github.com/aws-powertools/powertools-lambda-typescript/assets/7353869/ca3cc9da-7319-47a2-b787-29b6b99756db">

Reply:
tooltip hunter